docker/Dockerfile.python

```diff
@@ -0,0 +1,108 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+###
+#
+# The image to python package of GitHub/nvidia/spark-rapids-ml
+#
+# Arguments: CUDA_VERSION=11.5.x (CUDA Toolkit(>=11.5))
+# Arguments: GCC_VERSION=9 (gcc(>=9.3))
+# Arguments: CMAKE_VERSION=3.20.x (cmake(>=3.20))
+# Arguments: NINJA_VERSION=1.10.x (ninja(>=1.10))
+# Arguments: MAVEN_VERSION=3.8.6 (maven(>=3.3.9))
+#
+# Example to build & upload image :
+#    docker build -t spark-rapids-ml:cuda11.5.2 . -f Dockerfile \
+#        --build-arg CUDA_VERSION=11.5.2 \
+#        --build-arg GCC_VERSION=9 \
+#        --build-arg CMAKE_VERSION=3.23.3 \
+#        --build-arg NINJA_VERSION=1.10.2 \
+#        --build-arg MAVEN_VERSION=3.8.6
+#
+###
+
+ARG CUDA_VERSION=11.5.2
+FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu20.04
```

Comment:
Same comment about devel base as for other file.

Reply:
See reply above.